src/pages/SettingsPage.js

```diff
@@ -94,6 +120,29 @@ const SettingsPage = ({
                             Sign Out
                         </Text>
                     </TouchableOpacity>
+                    <View style={[styles.settingsPageBody, styles.mt6]}>
+                        <Text style={[styles.settingsPageHeading, styles.mt2, styles.mb4]} numberOfLines={1}>
+                            Chats
+                        </Text>
+                        <Text style={[styles.textP]}>
+                            How should we display chats on your home screen?
+                        </Text>
+                        <View style={[styles.mt2, styles.mb2]}>
+                            <RNPickerSelect
+                                onValueChange={updatePriorityMode}
+                                items={Object.values(priorityModes)}
+                                style={{
+                                    ...styles.picker,
```

Comment:
Can we just make this `style={styles.picker}`?

Reply:
yes. Thanks, I was merging extra properties there for ease while developing. I missed it on the cleanup 👍